xml2rfc_compat/serializers/__init__.py

```diff
@@ -64,6 +64,14 @@ def serialize(
     if anchor:
         root.set('anchor', anchor)
 
+    # Fill in appropriate target
+    try:
+        target = get_suitable_target(as_list(item.link or []))
```

Comment:
It might be worth adding tests for `referencegroup` references to ensure the code is doing what it should.